feat(agents): add AGENTS.md template (#221)
getoutreach/stencil-base: docs/agents/stencil-commands.md

# Stencil Commands

> Stencil commands are exposed as `mise` tasks. To list all available stencil tasks:
> 
> ```bash
> mise tasks | grep stencil
> ```

Reviewer: suggestion: this takes up too much context and should also be dependent upon `mise` help
```suggestion
Stencil commands are exposed as `mise` tasks, prefixed with `stencil`.
```

## Common tasks

| Task | Description |
|---|---|
| `stencil` | Run Stencil without updating module versions |
| `stencil:upgrade` | Run Stencil and update module versions if available |

> | `stencil:post` | Run all post-Stencil tasks |

Reviewer: suggestion: `stencil:post` is not normally meant to be run manually, it is always run after `stencil` or `stencil:upgrade`.
```suggestion
```
